pkg/network/usm/compile_test.go

```diff
@@ -18,13 +18,13 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/kernel"
 )
 
-func TestHttpCompile(t *testing.T) {
+func TestUSMCompile(t *testing.T) {
 	if !rtcHTTPSupported(t) {
```

Comment:
```suggestion
	if !rtcUSMSupported(t) {
```

Reply:
no, that's inaccurate as it check the kernel meets the HTTP monitoring requirements